eo-maven-plugin/src/test/java/org/eolang/maven/HomeTest.java

```diff
@@ -80,4 +83,22 @@ void existsInDirTest(@TempDir final Path temp) throws IOException {
             Matchers.is(true)
         );
     }
+
+    @Test
+    void pathModifyingTest() throws InvocationTargetException,
```

Comment:
one more code style comment. usually test names are written like "doesSomethingTest", for example "modifiesPathTest"